cli/ops/dispatch_minimal.rs

```diff
@@ -72,21 +69,23 @@ fn test_parse_min_record() {
   assert_eq!(parse_min_record(&buf), None);
 }
 
-pub fn dispatch_minimal(
+pub type MinimalOp = dyn Future<Item = i32, Error = ErrBox> + Send;
+pub type Dispatcher = fn(i32, Option<PinnedBuf>) -> Box<MinimalOp>;
```

Comment:
I'm not a fan of this name. I always imagined "dispatcher" as a function which maps "op id" to "op handler".

Reply:
that's what this does (modulo the op id part)

Reply:
Mmmm, I'm not sure. It looks like `Dispatcher` is actually an "op", it returns `MinimalOp` which is the result of that op. I had a feeling that it was supposed to be that `Dispatcher` selects `MinimalOp` which returns some `MinimalOpResult`. But that's nomenclature... just thinking out loud

Reply:
Op = Promise = Future .. there are many ops. An OpDispatcher (yet to be defined but outlined here https://github.com/denoland/deno/issues/2730) is something that returns an Op.

So I think the term Dispatcher is appropriate here. That said, I consider this bit ephemeral as I plan on adding the OpDispatcher trait soon.